From: Ondrej Jirman <megi@xff.cz>
Date: Mon, 27 Jun 2022 18:45:09 +0200
Subject: arm64: dts: rockchip: rk3399-pinephone-pro: Support both Type-C plug
 orientations

This fixes USB 3.0 and Alt-DP modes in reverse plug orientation.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index ea2e16d..e3a7073 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -1515,6 +1515,7 @@ &usbdrd3_0 {
 &usbdrd_dwc3_0 {
 	dr_mode = "otg";
 	extcon = <&typec_extcon_bridge>;
+	snps,usb3-phy-reset-quirk;
 	status = "okay";
 };
 
